lib/ReactViews/Tools/Tool.tsx

```diff
@@ -69,16 +69,16 @@ export class ToolButtonController extends MapNavigationItemController {
     return ViewerMode.Cesium;
   }
 
+  // TODO: The intention was for this to be translated in the getter. This is now removed as it was an incorrect use of useTranslationIfExists()
   get name() {
-    return useTranslationIfExists(this.props.toolName);
+    return this.props.toolName;
   }
+
+  // TODO: The intention was for this to be translated in the getter. This is now removed as it was an incorrect use of useTranslationIfExists()
   @computed
   get title() {
     const buttonState = this.active ? "open" : "closed";
-    return useTranslationIfExists(`tool.button.${buttonState}`, {
-      toolName: this.name,
-      toolNameLowerCase: this.name.toLowerCase()
-    });
+    return `tool.button.${buttonState}`;
```

Comment:
@nf-s the issue here is that applyTranslationifExists will only translate anything that has "translate#" at the start of the string otherwise it will just return the passed in value. So its not using applyTranslationIfExists in the right way. Should I still leave it in if its doing nothing?

Reply:
Nah you might be right @nf-s  Ill just check with @na9da about his intention for this part he had some ideas

Reply:
In my latest debugging, I think `name` and `title` doesn't get used anywhere. Will clean it up sometime - https://github.com/TerriaJS/terriajs/issues/6368